### DISCUSSION TITLE
Graph and Grid not being plotted

### DISCUSSION THREAD

**Discussion by nicyscaria** [3b1b/manim#1673] [Q&A] [answered]
Graphs and grids are not appearing in the manim videos generated. What could be the reason for this?

**Comment by TonyCrane** [comment, marked as answer]
See https://github.com/3b1b/manim/discussions/1546

**Reply by nicyscaria** [reply]
I am able to see circles and other geometric objects. It's only the x and y axes of the graphs, the graphs o be plotted and the grids that are not visible.
